data/themes/classic.json

```diff
@@ -36,7 +36,7 @@
   },
   "button": {
     "disabled-text": "#d5d5d5",
-    "disabled-bg": "#f5f5f5"
+    "disabled-bg": "#ededed"
   },
```

Comment:
Made this slightly darker as it was exactly the same color as the NCJ form background and you couldn't see the button when it was disabled. Have seen this in a few places so will sort them out as well.